webui/react/src/services/types.ts

```diff
@@ -6,6 +6,10 @@ export interface ApiCommonParams {
   cancelToken?: CancelToken,
 }
 
+export interface OptionParams {
```

Comment:
I'd see this as being similar to ApiCommonParams with the caveat that they are different in how they carry a cancel signal. since we only have this one param right now we can probably name them however we want.

Reply:
updated to `FetchOptionParams` to provide a bit more detail